infra/src/main/kotlin/fi/fta/geoviite/infra/common/ChangeTimeController.kt

```diff
@@ -22,6 +22,7 @@ data class CollectedChangeTimes(
     val layoutSwitch: Instant,
     val layoutKmPost: Instant,
     val geometryPlan: Instant,
+    val project: Instant,
```

Comment:
Projektikäsite liittyy geometriaan ja osa funktioista on geometriatoiminnallisuuden kanssa samassa paikassa. Projekteihin viitatessa ei ole toisaalta muualla käytetty geometry-etuliitettä, joten tämän PR:n yhteydessä tehdyissä muutoksissa etuliitettä (as in geometryProject tms.) ei vastaavasti käytetty. 

Muuttujien/funktioiden sijoittelullla pyrittiin kuitenkin tukemaan tätä liitännäisyyttä yleisemmissä paikoissa.